readinglist/tests/support.py

```diff
@@ -109,6 +109,13 @@ def test_invalid_record_raises_error(self):
                                   status=400)
         self.assertIn('errors', resp.json)
 
+    def test_create_with_invalid_json(self):
+        body = "{'foo': 'bad json'}"
```

Comment:
We should use something more obvious than the invalid `'`, because nobody is aware of that. Adding a missing closing bracket or something like that would be more effective.